src/services/store/events/Slices.ts

```diff
@@ -34,7 +34,9 @@ const eventsSlice = createSlice({
     },
     getEventListLoopStop: state => state,
     setSearchQuery: (state, { payload }) => {
-      if (payload.searchQuery.length) {
+      if (payload.searchQuery === 'clear') {
```

Comment:
Why do you use .searchQuery === 'clear' for clear searching?  you already have clearSearchQuery action for this!

Reply:
return previous logic back and use clearSearchQuery action for clear search.